digitalaidseattle/seattle-humane-app
# 1.2.2.x/input form save

## This PR is built on top of #49 and #48. 
Those should be merged first as this PR includes those changes.

## What type of PR is this? (check all applicable)


## Description

Fixes remaining issues to enable the E2E input form save / new service request/ticket scenario. 

## Related Tickets & Documents

[1.2 User creates request](https://projects.digitalaidseattle.org/features-details?recordId=recFJrAV8KeddrkLu)


## QA Instructions, Screenshots, Recordings

1. Install, build and run the client: 'yarn install && yarn dev`
1. Click new request
1. Fill in the all the information
1. Click Save
1. Verify form closes 
1. Check supabase for newly created records

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

## Added/updated tests?
_We are aiming for 100% statement coverage._

      have not been included_

## [optional] What gif best describes this PR or how it makes you feel?

![whoa, we're halfway there](https://media1.tenor.com/m/Cuf0ngxB-egAAAAC/halfway-there-bon-jovi.gif)
